addons/website_theme_install/models/theme_models.py

```diff
@@ -58,6 +61,22 @@ def _convert_to_base_model(self, website, **kwargs):
 
         return new_view
 
+    # TODO move to ir.module.module to factorize
+    def _post_process_creation(self, new_rec):
+        self.ensure_one()
+        IrTrans = self.env['ir.translation']
+        translations = IrTrans.search([
+            ('name', '=', 'theme.ir.ui.view,arch'),
+            ('type', '=', 'model_terms'),
+            ('res_id', '=', self.id),
+        ])
+        for trans in translations:
+            # TODO insert multi
```

Comment:
maybe it is the good timing to do the todo ;) ?

insert into (x,y,z)
select x,y,z from ir_translations where id in translations.ids ?

Reply:
Before spending more time on it, I wanted to be sure it was something we were actually going to do. If you agree on the idea of postprocessing, I will work on it to implement todo and so on.